blackbox/docs/sql/geo.txt

```diff
@@ -190,6 +190,12 @@ your geographic data::
     +--------------------+
     SELECT 1 row in set (... sec)
 
+Until Crate version ``0.55.x`` it was possible to query a point within a
```

Comment:
Doesn't this only depend on the version a table was created with? It's still backword compatible for old tables in case of an ugprade